src/ansys/fluent/core/session.py

```diff
@@ -122,13 +124,22 @@ def id(self) -> str:
         """Return the session id."""
         return self.fluent_connection.id
 
-    def start_transcript(self) -> None:
+    def start_transcript(self, file_path: str = None) -> None:
         """Start streaming of Fluent transcript."""
-        self.fluent_connection.start_transcript()
+        if file_path:
+            if Path(file_path).exists():
+                os.remove(file_path)
+            self.execute_tui(f'(cx-start-transcript "{file_path}")')
```

Comment:
This is good for client and server on the same machine. As a user here, I would expect the file to be written to the specified path on my local machine, but that's not the case if the server is remote. So you really want to delegate to the fluent connection here. We can request it to write the transcript to the specified path. We could also provide the option to pass in a callback to receive the transcript strings.